locations/spiders/procolor_collision_us.py

```diff
@@ -0,0 +1,13 @@
+from locations.storefinders.wp_store_locator import WPStoreLocatorSpider
+
+
+class ProColorCollisionUSSpider(WPStoreLocatorSpider):
```

Comment:
There are some locations in Canada too so we should drop the `US` suffix for this spider name.